### PR TITLE
[ci-skip] add README.md files for each Helm chart

### LINKED ISSUE #2283: feat(fabric): update platform helm charts with README.md
### Description
As a developer I want to add proper documentation for the helm charts, before they are packaged and published on bevel helm repo

### Acceptance Criteria
1. README.md file added.
2. values.yaml file filled with default values.
3. chart.yaml versions updated.
.

### PR DESCRIPTION
**Pull request by saurabhkumarkardam**
### **Commit to be reviewed**
---

**docs(fabric): add README.md files for each Fabric Helm chart**

```
This commit adds the README.md file for each Fabric Helm chart. The README.md files provide essential information and instructions about the respective Helm charts, enabling users and developers to deploy each chart individually.

Additionally, the following changes have been made:

 • Updated values.yaml of each Helm chart with default values to provide a clear configuration reference.
 • Added helpers.tpl template file only to Helm charts where it was missing, ensuring consistency across the charts.
 • Updated the description and version fields of each Fabric Helm chart.

These changes enhance the usability and documentation of the Fabric Helm charts, making it easier for users and developers to understand and work with the charts.
```

fixes #2283

### CODE REVIEW

File: platforms/hyperledger-fabric/charts/peernode/values.yaml [hunk 1]
```diff
@@ -11,13 +11,13 @@ metadata:
...
-    couchdb: 
+    couchdb: hyperledger/fabric-couchdb:0.4.14
     #Provide the valid image name and version for fabric peer
     #Eg. hyperledger/fabric-peer:1.4.0
-    peer: 
+    peer: hyperledger/fabric-peer:1.4.0
     #Provide the valid image name and version to read certificates from vault server 
     #Eg. alpineutils: hyperledgerlabs/alpine-utils:1.0
-    alpineutils: 
+    alpineutils: hyperledgerlabs/alpine-utils:1.0
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-fabric/charts/peernode/values.yaml:20]: confirm what the code uses, should be ghcr.io/hyperledger, same for other charts using alpine

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-fabric/charts/generate_cacerts/Chart.yaml [hunk 2]
```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for ca server configuration
+description: "Hyperledger Fabric: Configures ca server."
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/generate_cacerts/Chart.yaml:9]: This should be a job to generate CA Server certs

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-fabric/charts/create_channel/values.yaml [hunk 3]
```diff
@@ -46,39 +46,39 @@ peer:
...
+  name: mychannel
 
 orderer:
   #Provide the address for orderer
   #Eg. address: orderer.fratest-com:7050
-  address:
+  address: 
 
 #Provide the base64 encoded file contents for channeltx
 channeltx:
```
[the remark this answers is not in the document]
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/create_channel/values.yaml]: Fixed.

File: platforms/hyperledger-fabric/charts/instantiate_chaincode/values.yaml [hunk 4]
```diff
@@ -11,10 +11,10 @@ metadata:
   images:
     #Provide the valid image name and version for fabric tools
     #Eg. fabrictools: hyperledger/fabric-tools:1.4.0
-    fabrictools: 
+    fabrictools: hyperledger/fabric-tools:1.4.0
     #Provide the valid image name and version to read certificates from vault server
     #Eg. alpineutils: gcr.io/acn-stp-on-blockchain/alpine-utils
-    alpineutils: 
+    alpineutils: gcr.io/acn-stp-on-blockchain/alpine-utils
```
[the remark this answers is not in the document]
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/instantiate_chaincode/values.yaml]: Fixed it across all the Helm charts.

File: platforms/hyperledger-fabric/charts/peernode/values.yaml (hunk 1 above)
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/peernode/values.yaml]: Fixed it across all the Helm charts.

File: platforms/hyperledger-fabric/charts/generate_cacerts/Chart.yaml (hunk 2 above)
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/generate_cacerts/Chart.yaml]: Fixed.

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**